plugins/touchegg/Main.vala

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2020 elementary, Inc (https://elementary.io)
+ *           2020 José Expósito <jose.exposito89@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+public class Gala.Plugins.Touchegg.Plugin : Gala.Plugin {
+    private Gala.WindowManager? wm = null;
+    private Client? client = null;
+    private GLib.Settings gala_settings = new GLib.Settings ("io.elementary.desktop.wm.gestures");
+    private GLib.Settings touchpad_settings = new GLib.Settings ("org.gnome.desktop.peripherals.touchpad");
```

Comment:
Is it possible to initialize these in `initialize`? I think this would be the more correct option here.